lib/toxiproxy.rb

```diff
@@ -68,6 +68,11 @@ def self.all
   # Sets the toxiproxy host to use.
   def self.host=(host)
     @uri = host.is_a?(::URI) ? host : ::URI.parse(host)
+    if defined? @http
+      @http.finish() if @http && @http.started?
+      @http = nil
+    end
```

Comment:
I think we could extract it into some sort of method to reset all connections and it might be useful in the tests too

I throw some ideas (from more suitable to less)

```ruby
  reset_http_client!
  # or
  reset_http!
  # or
  reset_http_connections!
```

Reply:
thank you! Now it looks cleaner ❤️